AppleStore/AppleStore/ItemWebPageViewController.swift

```diff
@@ -22,7 +22,7 @@ final class ItemWebPageViewController: UIViewController {
     private var shareButton: UIBarButtonItem!
     
     // MARK: Public Properties
-    var item: Item?
+    public var item: Item?
```

Comment:
```suggestion
    var item: Item?
```
никогда не пишем public